packages/time-picker/test/events.test.js

```diff
@@ -8,145 +8,6 @@ import '../vaadin-time-picker.js';
 describe('events', () => {
   let timePicker;
 
-  describe('change event', () => {
-    let changeSpy, inputElement;
-
-    beforeEach(() => {
-      timePicker = fixtureSync(`<vaadin-time-picker></vaadin-time-picker>`);
-      inputElement = timePicker.inputElement;
-      changeSpy = sinon.spy();
-      timePicker.addEventListener('change', changeSpy);
-    });
-
-    it('should be fired when committing user input with Enter', async () => {
-      inputElement.focus();
-      await sendKeys({ type: '00:00' });
-      await sendKeys({ press: 'Enter' });
-      expect(changeSpy.calledOnce).to.be.true;
-    });
-
-    it('should be fired when selecting a time with arrows and committing with Enter', () => {
-      arrowDown(inputElement);
-      arrowDown(inputElement);
-      enter(inputElement);
-      expect(changeSpy.calledOnce).to.be.true;
-    });
-
-    it('should be fired on clear button click', () => {
-      timePicker.clearButtonVisible = true;
-      timePicker.value = '00:00';
-      timePicker.$.clearButton.click();
-      expect(changeSpy.calledOnce).to.be.true;
-    });
-
-    it('should be fired on arrow keys when no dropdown opens', () => {
-      timePicker.step = 1;
-      arrowDown(inputElement);
-      expect(changeSpy.calledOnce).to.be.true;
-      arrowUp(inputElement);
-      expect(changeSpy.calledTwice).to.be.true;
-    });
-
-    it('should be fired after value-changed event on arrow keys', () => {
-      timePicker.step = 0.5;
-      const valueChangedSpy = sinon.spy();
-      timePicker.addEventListener('value-changed', valueChangedSpy);
-      arrowDown(inputElement);
-      expect(valueChangedSpy.calledOnce).to.be.true;
-      expect(changeSpy.calledOnce).to.be.true;
-      expect(valueChangedSpy.calledBefore(changeSpy)).to.be.true;
-    });
-
-    it('should not be fired on focused time change', async () => {
-      inputElement.focus();
-      await sendKeys({ type: '00:00' });
-      expect(changeSpy.called).to.be.false;
-    });
-
-    it('should not be fired on programmatic value change', () => {
-      timePicker.value = '01:00';
-      expect(changeSpy.called).to.be.false;
-    });
-
-    it('should not be fired on programmatic value change after manual one', () => {
```

Comment:
**note:** It's a very old test and it's no longer clear what part of the code it aims to cover, especially given that nowadays `time-picker` doesn't have any logic that could potentially lead to a change event on programmatic value change. I guess it's enough to have this [simple test](https://github.com/vaadin/web-components/pull/6416/files#diff-6f732b88f4b5dee8efc2e4bc89c04d24c79cd71d05a48e4ce894455bcff8f8dcR257-R259) ensuring that the change event isn't fired on programmatic value change, without any preceeding actions.